config.ts

```diff
@@ -0,0 +1,180 @@
+const siteConfigData = {
```

Comment:
Is there a way we can do away with this file? I know it's a bit of extra work but doing something [like this](https://github.com/prisma/docs/blob/docusaurus/src/components/HomepageFeatures/index.tsx) feels a bit better to me.

Reply:
To have content + logic on the same file? sure!